qiskit_ionq/helpers.py

```diff
@@ -389,8 +395,8 @@ def qiskit_to_ionq(circuit, backend, passed_args=None):
     }
     if target == "simulator":
         ionq_json["noise"] = {
-                "model": backend.options.noise_model,
-                "seed": backend.options.sampler_seed,
+            "model": passed_args.get("noise_model", backend.options.noise_model),
```

Comment:
Seems to be the most significant material change here. Only word of caution/nit-suggestion is to: `passed_args.get("noise_model", None) or backend.options.noise_model)`, to protect against cases where `passed_args["noise_model"]` exists but is for Falsey or otherwise invalid.

Reply:
Sounds good I'll update that 👍